Makefile

```diff
@@ -107,5 +107,4 @@ update-test-deps: update-deps test
 
 .PHONY: integration-test
 integration-test:
-	go test -c -tags=integration -o dao_tests.test ./cmd/dao_tests
-	./dao_tests.test
+	go test -tags=integration ./cmd/dao_tests
```

Comment:
I created a symlink which allows running tests just like that, looks cleaner to me.

Reply:
I'm bit lost here, why is this change even here, it feels like we are trying to address bit too much for a single PR.
I'm ok with the change, but it seems like bit out of scope of the PR :thinking: